pyccel/codegen/printing/ccode.py

```diff
@@ -1200,6 +1200,13 @@ def _print_MathTrunc(self, expr):
     def _print_FunctionAddress(self, expr):
         return expr.name
 
+    def _print_NumpyWhere(self, expr):
+        cond = self._print(expr.condition)
+        value_true = self._print(expr.x)
+        value_false = self._print(expr.y)
+        stmt = '{cond} ? {true} : {false}'.format(cond = cond, true =value_true, false = value_false)
```

Comment:
I think the names `value_true` and `value_false` used here are much clearer than `x` and `y`. Could we change the definition of the `NumpyWhere` class to use these names?